cfg_mgmt/util.py

```diff
@@ -494,25 +494,46 @@ def process_cfg_queue_and_persist_in_repo(
     return True
 
 
+def _policy_violation_for_issue_type(issue_type: str) -> cmm.CfgStatusEvaluationAspects:
+    if issue_type == gci._label_no_responsible:
+        return cmm.CfgStatusEvaluationAspects.NO_RESPONSIBLE
```

Comment:
as stated before, please rename enum. for example to just "CfgElementPolicyViolation"

Reply:
Please see https://github.com/gardener/cc-utils/pull/784#discussion_r1038124370.
I'd prefer to do this in a separate PR.